Tasks/Maven/task.json

```diff
@@ -80,12 +93,23 @@
             "groupName":"advanced",
             "defaultValue":"default",
             "helpMarkDown": "Will attempt to discover the path to the selected JDK version and set JAVA_HOME accordingly.",
+            "visibleRule":"javaHomeSelection = JDKVersion",
             "options": {
                 "default":"default",
                 "1.8":"JDK 8",
                 "1.7":"JDK 7",
                 "1.6":"JDK 6"
             }
+        },     
+        {
+            "name":"jdkUserInputPath",
+            "type":"string",
+            "label":"JDK Path",
+            "required":true,
+            "groupName":"advanced",
+            "defaultValue":"",
+            "helpMarkDown": "Sets JAVA_HOME to the given path.",
+            "visibleRule":"javaHomeSelection = Path"
         },
         { 
             "name": "jdkArchitecture", 
```

Comment:
There is also a generated version of the Task.json that includes localized keys. It should be generated as part of the gulp build. I think that file still has to be checked in, but check with Eric for details.